app/src/main/java/org/sil/gatherwords/SettingsWordFieldSetupFragment.java

```diff
@@ -0,0 +1,108 @@
+package org.sil.gatherwords;
+
+import android.content.Context;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Activities that contain this fragment must implement the
+ * {@link SettingsWordFieldSetupFragment.OnFragmentInteractionListener} interface
+ * to handle interaction events.
+ * Use the {@link SettingsWordFieldSetupFragment#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class SettingsWordFieldSetupFragment extends Fragment {
+    // TODO: Rename parameter arguments, choose names that match
+    // the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+    private static final String ARG_PARAM1 = "param1";
+    private static final String ARG_PARAM2 = "param2";
+
+    // TODO: Rename and change types of parameters
+    private String mParam1;
+    private String mParam2;
+
+    private OnFragmentInteractionListener mListener;
+
+    public SettingsWordFieldSetupFragment() {
+        // Required empty public constructor
+    }
+
+    /**
+     * Use this factory method to create a new instance of
+     * this fragment using the provided parameters.
+     *
+     * @param param1 Parameter 1.
+     * @param param2 Parameter 2.
+     * @return A new instance of fragment SettingsWordFieldSetupFragment.
+     */
+    // TODO: Rename and change types and number of parameters
+    public static SettingsWordFieldSetupFragment newInstance(String param1, String param2) {
+        SettingsWordFieldSetupFragment fragment = new SettingsWordFieldSetupFragment();
+        Bundle args = new Bundle();
+        args.putString(ARG_PARAM1, param1);
+        args.putString(ARG_PARAM2, param2);
+        fragment.setArguments(args);
+        return fragment;
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        if (getArguments() != null) {
+            mParam1 = getArguments().getString(ARG_PARAM1);
+            mParam2 = getArguments().getString(ARG_PARAM2);
```

Comment:
What is the purpose of these parameters?  You store and load them, but I do not see them used.